pkg/upload/js.go

```diff
@@ -0,0 +1,40 @@
+package upload
+
+import (
+	"fmt"
+
+	"github.com/bugsnag/bugsnag-cli/pkg/log"
+	"github.com/bugsnag/bugsnag-cli/pkg/utils"
+)
+
+type JsOptions struct {
+	VersionName string      `help:"The version of the app that the source map applies to (as set in the JavaScript notifier). Defaults to the version in the package.json file."`
+	BundleUrl   string      `help:"The url of the minified JavaScript file that the source map relates to. Asterisks can be used as a wildcard."`
+	SourceMap   string      `help:"Path to the source map file. This usually has the .min.js extension." type:"path"`
+	Bundle      string      `help:"Path to the minified JavaScript file that the source map relates to. If this is not provided then the file will be obtained when an error event is received." type:"path"`
+	ProjectRoot string      `help:"The root path of the project on the filesystem where the source map was built. This will be stripped off from the file name in the displayed stacktraces." type:"path"`
+	Path        utils.Paths `arg:"" name:"path" help:"Path to directory or file to upload" type:"path" default:"."`
```

Comment:
Although it makes sense to allow a directory and use file name conventions to find all the bundle/map pairs, I think the use case for specifying a single file isn't clear and will involve a degree of logic to deduce if it's the bundle or map.

Shall we only allow directories here?

```suggestion
	Path        utils.Paths `arg:"" name:"path" help:"Path to a directory of source maps and bundles to upload" type:"path" default:"."`
```